ecosystem/nft-metadata-crawler-parser/migrations/2023-06-27-001532_create_tables/down.sql

```diff
@@ -1,2 +1,3 @@
 DROP TABLE IF EXISTS nft_metadata_crawler.parsed_token_uris;
+DROP TABLE IF EXISTS ledger_infos;
```

Comment:
wait this needs to say `nft_metadata_crawler.ledger_infos`

You should always run DATABASE=... diesel migration redo

Reply:
ah in this case it'll still pass...it's just a typo i guess.